examples/advanced/quantum_volume.py

```diff
@@ -104,6 +104,41 @@ def compute_heavy_set(circuit: cirq.Circuit) -> List[int]:
     ]
 
 
+def sample_heavy_set(circuit: cirq.Circuit,
+                     heavy_set: List[int],
+                     *,
+                     sampler: cirq.Sampler = cirq.Simulator()) -> float:
+    """Run a sampler over the given circuit and compute the percentage of its
+       outputs that are in the heavy set.
+
+    Args:
+        circuit: The circuit to sample.
+        heabv_set: The previously-computed heavy set for the given circuit.
+        sampler: The sampler to run on the given circuit.
+
+    Returns:
+        A probability percentage, from 0 to 1, representing how many of the
+        output bit-strings were in the heaby set.
+
+    """
+    # Add measure gates to the end of the circpuit.
+    circuit_copy = circuit.copy()  # So we don't mutate the original circuit.
```

Comment:
`circuit + cirq.measure(...)`

Reply:
Done, although TBH I'm not sure it's cleaner :\

Reply:
Oh right, adding the operation directly still doesn't work. Bah. Whichever.

Reply:
Use `cirq.Circuit(` instead of `cirq.Circuit.from_ops(`

Reply:
Ooh actually you can add a list of gates to the circuit, I wrapped it in brackets and it worked :) Thanks!

Reply:
After https://github.com/quantumlib/Cirq/pull/2189 merges you can sync with master and add the operation without the wrapping list.

Reply:
Sweet, thanks! Synced.